src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -679,12 +682,12 @@ def is_transition_rate(name: str) -> bool:
 
 def match_rr_to_cause_name(data: Union[str, pd.DataFrame], source_key: EntityKey):
     # Need to make relative risk data match causes in the model
-#    is_calculated_paf = (
-#        (source_key == data_keys.LDL_C.PAF)
-#        or (source_key == data_keys.SBP.PAF)
-#        or (source_key == data_keys.BMI.PAF)
-#    )
     is_calculated_paf = False
+    # is_calculated_paf = (
+    #     (source_key == data_keys.LDL_C.PAF)
+    #     or (source_key == data_keys.SBP.PAF)
+    #     or (source_key == data_keys.BMI.PAF)
+    # )
```

Comment:
You moved this block only, but why is it commented?

Reply:
The current workflow requires this block to be commended/uncommented depending on whether you're generating pafs or not.